packages/server/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "scripts": {
     "build": "tsc",
+    "ci:build": "NODE_ENV=test node ../../node_modules/typescript/lib/tsc",
```

Comment:
why can't we just use `tsc`?

Reply:
We need typescript on our docker image to build the code but after running `yarn`, we'll have a copy of this package already installed using the version of ts that we use in the proj. So instead of doing something like 

```yarn add global typescript```

we just use the tsc that we already have.

I actually tried to `yarn add global typescript` as one of the commands to run in the init stage of each job, but that won't work because the job doesn't have the right permissions to modify the docker container directly.

The other alternative would be to write our own `Dockerfile` and [tell circle to use that](https://circleci.com/docs/2.0/custom-images/). I kind of didn't want to do that because it seemed a bit unnecessary just for tsc. But it would be nice to have! Feel free to make a PR if you want!

Reply:
Hmm but we should be able to reference the local tsc install the same way as we do in the build script. Instead of using the relative path to it in node_modules

Reply:
I don't think it needs to be installed globally. I'm wondering why we need to use the relative path in this case
https://docs.npmjs.com/misc/scripts#path

Reply:
Maybe the error I got had something to do with the monorepo setup. The TS package is hoisted to root, so maybe that's why? I have an idea of what we can try. Will follow up with a PR